packages/host/tests/integration/search-index-test.ts

```diff
@@ -3588,5 +3588,18 @@ posts/ignore-me.json
         [`${paths.url}books/1`]
       );
     });
+
+    test(`can search for cards by using the 'contains' filter`, async function (assert) {
+      let { data: matching } = await indexer.search({
+        filter: {
+          contains: { title: 'ca' },
+        },
```

Comment:
generally all predicates need to include an `on` property that indicates which card definition the field (in this case `title`) belongs to. I think since all cards have a `title` field you are able to get away without having an `on`, but please do include a test that uses `on` for a field that is specific to a particular card type